### ISSUE TITLE
Add support for django-unfold

### ISSUE THREAD

**Issue by twFroggen** [jrief/django-admin-sortable2#398]
First and foremost, I would like to extend my gratitude for developing such an impressive function.

I am writing to report a compatibility issue I encountered with the django-unfold when integrated with the sortable2 interface. It seems that the styles and action buttons from Sortable2 do not align well with the Unfold design.

![image](https://github.com/jrief/django-admin-sortable2/assets/138838525/fe58a511-fad7-44da-9e37-ce723fb9778a)

Could you please consider enhancing the compatibility between these tools in your future updates? Any assistance or guidance in this matter would be greatly appreciated.

Thank you for your attention to this matter. Looking forward to your favorable response.

Best regards,
Froggen

**Comment by jrief** [comment]
Feel free to create a pull request. I currently have no need for django-unfold but I'm open for enhancements to this library.

**Comment by Investigamer** [comment]
@twFroggen Encountered this same issue, looks like a few people out there have. I managed to solve this issue by extending the `SortableAdminMixin` class and patching in a modified `action_form` class attribute to replace the malformed one.

### Original `MovePageActionForm`
```py
class MovePageActionForm(admin.helpers.ActionForm):
    step = IntegerField(
        required=False,
        initial=1,
        widget=widgets.NumberInput(attrs={'id': 'changelist-form-step'}),
        label=False
    )
    page = IntegerField(
        required=False,
        widget=widgets.NumberInput(attrs={'id': 'changelist-form-page'}),
        label=False
    )
```

### `MovePageActionForm` Patched with Unfold Classes
```
# from unfold.forms import ActionForm
class UnfoldMovePageActionForm(ActionForm):
    # from django.forms import IntegerField
    step = IntegerField(
        required=False,
        initial=1,
        # from unfold.widgets import UnfoldAdminIntegerFieldWidget
        widget=UnfoldAdminIntegerFieldWidget(attrs={'id': 'changelist-form-step'}),
        label=False
    )
    page = IntegerField(
        required=False,
        widget=UnfoldAdminIntegerFieldWidget(attrs={'id': 'changelist-form-page'}),
        label=False
    )
    
# Inject it in a extended class
class UnfoldSortableMixin(SortableAdminMixin):
    action_form = UnfoldMovePageActionForm

# Use it in your model admin
@admin.register(MyModel)
class MyModelAdmin(UnfoldSortableMixin, ModelAdmin[Source]):
    # ... stuff here ...
```

**Comment by Disaxy** [comment]
@Investigamer Hello! I faced the same problem, but when using inlines, styles went, can you tell me how to correct that the styles were from django unfold?

![Image](https://github.com/user-attachments/assets/9efde23f-3a34-43e1-8360-83e97f4b4185)

**Comment by jrief** [comment]
> [@Investigamer](https://github.com/Investigamer) Hello! I faced the same problem, but when using inlines, styles went, can you tell me how to correct that the styles were from django unfold?

I don't know. I never have used django-unfold.

**Comment by Disaxy** [comment]
Hello Django Unfold already supports such a [sorting](https://unfoldadmin.com/docs/inlines/sortable/) out of the box.

**Comment by Investigamer** [comment]
> Hello, [@Disaxy](https://github.com/Disaxy)
> 
> I have noticed that the drag-and-drop functionality is also not working in the latest version. Could you please advise on a solution for this issue? Thank you.
> 
> Unfold version: 0.51.0 Sortable2 version: 2.2.4

Yes unfortunately my previous monkey patch is no longer working anymore :/

@Disaxy unfortunately their sorting feature is only supported for inlines, which I do not use and have no interest in using. Hopefully someone comes along with an idea here, I will continue to work (frustratingly) on a solution.
